workspace/utilities/site-imports.xsl

```diff
@@ -56,7 +56,7 @@
 	<xsl:import href="lib/util-auto-handle.xsl" />
 	<xsl:import href="lib/util-rewrite-markdown.xsl" />
 	<xsl:import href="lib/util-transition-scroll-attr.xsl" />
-	<xsl:import href="lib/util-ficelle.xsl" />
+	<!-- <xsl:import href="lib/util-ficelle.xsl" /> -->
 	<xsl:import href="lib/util-count.xsl" />
```

Comment:
I do not see util-picture being imported ?
Or I am missing something ? We need a default implementation of picture right ?

Reply:
what ?

Reply:
this is not related to picture.

Reply:
ok we figured it out, picture is imported in master.xsl